### PR TITLE
* Docs: Add docs about enabling and disabling encryption

### PR DESCRIPTION
**Pull request by stgarf**
Adding docs outlining basic syntax of enabling and disabling encryption of your paperless document installation.

### CODE REVIEW

**Comment by stgarf** [comment]
Hrrmmmm, maybe a comment will re-trigger Travis?

**Comment by stgarf** [comment]
@bmsleight  Looks like this just needs a second set of eyes before merging?

**Comment by stgarf** [comment]
@jat255 Hmm :( looks like Travis isn't actually doing any CI on this PR https://travis-ci.org/the-paperless-project/paperless is it still in a weird state?

**Comment by jat255** [comment]
Yeah, looks like #512 will need to be resolved to get PRs working again. I'm not overly familiar with travis, but I'll try to take a look.

**Comment by stgarf** [comment]
It "works" for a limited set of the meaning of the word "works" as it looks like #519 was tested through CI correctly... https://travis-ci.org/the-paperless-project/paperless/builds/500987986

**Comment by stgarf** [comment]
I think closing/re-opening this PR or closing/opening a new PR may fix the problem here... trying.

**Comment by stgarf** [comment]
Hey Travis, if you're feeling like "working", I'm attempting to re-open this PR... go ahead and check it out :+1:

**Comment by stgarf** [comment]
(boom) https://travis-ci.org/the-paperless-project/paperless/builds/501170342

**Comment by jat255** [comment]
@stgarf I don't understand what sort of black magic that was, but glad you got it working!

**Comment by stgarf** [comment]
@jat255  IT just looks like Travis was in a stale state from the conversion of the repo to an organization. Simply closing and re-opening this PR was enough to re-trigger Travis. That's all that's likely needed on the other two or three PR's that I see are stalled as well (hint hint ;)).

File: docs/utilities.rst [hunk 1]
```diff
@@ -216,3 +216,62 @@ That's it.  It'll loop over all of the documents in your database and attempt
...
+      --passphrase PASSPHRASE
+                            If PAPERLESS_PASSPHRASE isn't set already, you need to
+                            specify it here
+
+Enabling Encryption
+...................
+
+Basic usage to enable encryption of your document store (**USE A MORE SECURE PASSPHRASE**):
+
+(Note: If PAPERLESS_PASSPHRASE isn't set already, you need to specify it here)
```
Comment (pitkley) [inline docs/utilities.rst]:
```suggestion
(Note: If ``PAPERLESS_PASSPHRASE`` isn't set already, you need to specify it here)
```

File: docs/utilities.rst [hunk 2]
```diff
@@ -216,3 +216,62 @@ That's it.  It'll loop over all of the documents in your database and attempt
...
+
+    $ /path/to/paperless/src/manage.py change_storage_type [--passphrase SECR3TP4SSPHRA$E] unencrypted gpg
+
+
+Disabling Encryption
+....................
+
+Basic usage to enable encryption of your document store
+
+(Note: Again, if PAPERLESS_PASSPHRASE isn't set already, you need to specify it here)
```
Comment (pitkley) [inline docs/utilities.rst]:
```suggestion
(Note: Again, if ``PAPERLESS_PASSPHRASE`` isn't set already, you need to specify it here)
```

File: docs/utilities.rst [hunk 3]
```diff
@@ -216,3 +216,62 @@ That's it.  It'll loop over all of the documents in your database and attempt
...
+
+.. code:: bash
+
+    $ /path/to/paperless/src/manage.py change_storage_type [--passphrase SECR3TP4SSPHRA$E] unencrypted gpg
+
+
+Disabling Encryption
+....................
+
+Basic usage to enable encryption of your document store
```
Comment (pitkley) [inline docs/utilities.rst]:
```suggestion
Basic usage to enable encryption of your document store:
```

**Pull request merged**